### DISCUSSION TITLE
Having issues with emojis and text boxes

### DISCUSSION THREAD

**Discussion by AGWiki2021** [requarks/wiki#4220] [Help / Questions]
Hi all,

I have an issue placing emojis on my Wiki.JS site. I'm trying to put ": droplet :" (I didn't put the spaces in there) on my website however it's not turning into a emoji. I have no idea on what to do. Also, I'm trying to add some link boxes to my website. Check the first image on my idea on what I intend. I also want to put some information boxes into my website. See 2nd picture for info. I also would like to make a border around my images.

Thanks in advance

![Capture](https://user-images.githubusercontent.com/87502232/126424517-3d74657f-ad80-4f1b-8785-da11b649185a.PNG)

![Capture2](https://user-images.githubusercontent.com/87502232/126424531-713d7618-4136-4734-9b9b-64a86d8cda3c.PNG)

**Comment by benjamincarmel** [comment]
Can you share the wiki url?

**Reply by AGWiki2021** [reply]
Which site? The examples I just took from or my site?
